source/glue/normalizers/phone_normalizer.py

```diff
@@ -0,0 +1,39 @@
+import phonenumbers
+
+restrictedDigits = {
+    "GB": ["4", "6"]
+}
+
+def check_if_valid(number: int, countryCode: str):
+    if countryCode in restrictedDigits:
```

Comment:
Unless am wrong `countryCode` starts at the beginning of every phone number.

```suggestion
    if restrictedDigits.startswith(f"+{countryCode}"):
```

Reply:
this code is checking if the passed in countryCode has a digit restriction (i.e. US does not, but other countries might in the future like GB)

Reply:
ah ok.